src/shogun/machine/LinearMachine.cpp

```diff
@@ -103,6 +111,16 @@ float64_t CLinearMachine::get_bias()
 	return bias;
 }
 
+void CLinearMachine::set_bias_flag(bool f)
```

Comment:
minor: we usually have the same name as the member variable in the parameter of a method, i.e. bias_flag when member is m_bias_flag